src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -393,6 +393,7 @@ def transform_summary(self,
             'group_by_organ',
             'terms',
             field='contents.cell_suspensions.organ.keyword',
+            meta={'path': ('contents', 'cell_suspensions', 'organ')},
```

Comment:
I had something even more generic in mind. Notice how `field` and `meta.path` are covariant. Why do I have to hard-code the derived value if it can be computed?